api/src/resource-extraction.js

```diff
@@ -23,7 +23,34 @@ const isAttachmentExtractable = name => {
 // Map of attachmentName -> attachmentDigest used to avoid extraction of unchanged documents
 let extractedDigests = {};
 
-const createFolderIfDne = x => !fs.existsSync(x) && fs.mkdirSync(x);
+const createFolderIfDne = folderPath => !fs.existsSync(folderPath) && fs.mkdirSync(folderPath);
+
+const removeDirectoryRecursive = (dirPath) => {
+  if (!fs.existsSync(dirPath)) {
+    return;
+  }
+
+  const files = fs.readdirSync(dirPath) || [];
+
+  files.forEach(fileName => {
+    const filePath = `${dirPath}/${fileName}`;
+
+    if (fs.statSync(filePath).isDirectory()) {
+      removeDirectoryRecursive(filePath);
+    } else {
+      fs.unlinkSync(filePath);
+    }
+  });
+
+  fs.rmdirSync(dirPath);
+};
+
+const removeDirectory = () => {
+  const outputPath = environment.getExtractedResourcesPath();
+
+  // ToDo: When migrating to node +v.12.10.0, replace with: fs.rmdirSync(outputPath, { recursive: true });
```

Comment:
Nit: 
this comment is deceiving, because it implies that we will "migrate" to node 12. 
I think it should say something along the lines of:  "When no longer supporting Node.js  -v.12"

Reply:
Fixed